app/ledger.js

```diff
@@ -967,7 +979,9 @@ var run = (delayTime) => {
       var result
       var siteSetting = siteSettings.get(`https?://${winner}`)
 
-      if ((siteSetting) && (siteSetting.get('ledgerPayments') === false)) return
+      if ((siteSetting) &&
```

Comment:
@bbondy - just so i'm on the same page: the change here doesn't break anything, i merely believe that it is unnecessary. there's no harm in keeping it...